packages/cli/src/commands/governance/view.ts

```diff
@@ -9,20 +10,26 @@ export default class View extends BaseCommand {
   static flags = {
     ...BaseCommand.flags,
     proposalID: flags.string({ required: true, description: 'UUID of proposal to view' }),
+    raw: flags.boolean({ required: false, description: 'Display proposal in raw bytes format' }),
```

Comment:
What's the default?

Reply:
defaults to false (equivalent to excluding as flag in oclif)